cypress.config.ts

```diff
@@ -57,7 +55,7 @@ export default defineConfig({
     },
     baseUrl: 'http://localhost:3007',
     excludeSpecPattern: '**/!(*.cy).ts',
-    specPattern: 'integration_tests/integration/**/*.cy.{js,jsx,ts,tsx}',
+    specPattern: 'integration_tests/e2e/**/*.cy.{js,jsx,ts,tsx}',
```

Comment:
I've renamed folder for Cypress tests to be `integration/e2e` instead of `integration/integration` so it matches template (and thus easier to compare and merge changes).